rfc8446.xml

```diff
@@ -4175,7 +4183,7 @@ the server system as a whole.</t>
 
 <t>The first class of attack can be prevented by sharing state to guarantee that
 the 0-RTT data is accepted at most once.  Servers SHOULD provide that level of
-replay safety, by implementing one of the methods described in this section or
+replay safety by implementing one of the methods described in this section or
```

Comment:
If I was author I would keep this comma, but I don't think my opinion is actually relevant in this case.